web/components/TopBar/ImportButton.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+  <v-dialog v-model="data.dialog" width="500">
+    <template #activator="{ on }">
+      <v-btn color="primary ma-2" v-on="on"> Import </v-btn>
+    </template>
+
+    <v-card>
+      <v-card-title class="text-h5 grey lighten-2"> Import </v-card-title>
+
+      <v-card-text>
+        WARNING: All current created resources will be deleted.
+        <form>
+          <input type="file" accept=".yml" @change="readfile" />
+        </form>
+      </v-card-text>
+
+      <v-divider></v-divider>
+
+      <v-card-actions>
+        <v-spacer></v-spacer>
+        <v-btn color="green darken-1" text @click="data.dialog = false">
+          Cancel
+        </v-btn>
+        <v-btn
+          color="green darken-1"
+          text
+          :disabled="data.isImportButtonDisabled"
+          @click="ImportScheduler()"
+        >
+          Import
+        </v-btn>
+      </v-card-actions>
+    </v-card>
+  </v-dialog>
+</template>
+
+<script lang="ts">
+import { defineComponent, inject, reactive } from "@nuxtjs/composition-api";
+import { importScheduler, ResourcesForImport } from "~/api/v1/export";
+import yaml from "js-yaml";
+import SnackBarStoreKey from "../StoreKey/SnackBarStoreKey";
+import PriorityClassStoreKey from "../StoreKey/PriorityClassStoreKey";
+import StorageClassStoreKey from "../StoreKey/StorageClassStoreKey";
+import PersistentVolumeClaimStoreKey from "../StoreKey/PVCStoreKey";
+import PersistentVolumeStoreKey from "../StoreKey/PVStoreKey";
+import NodeStoreKey from "../StoreKey/NodeStoreKey";
+import PodStoreKey from "../StoreKey/PodStoreKey";
+
+interface SelectedItem {
+  dialog: boolean;
+  filedata: ResourcesForImport;
+  isImportButtonDisabled: boolean;
+}
+
+export default defineComponent({
+  setup() {
+    const data = reactive({
```

Comment:
Add `watch` func and reset SelectedItem is `data.dialog` is changed. 

You can see how to use `watch` func in ResourceBar component
https://github.com/kubernetes-sigs/kube-scheduler-simulator/blob/master/web/components/ResourceBar/ResourceBar.vue#L210

Reply:
I fixed that problem.
But this was caused by that file data being left in the input tag of the HTML.
So, I initialize and solve it by used to `v-if` when the dialog is closed.

https://github.com/kubernetes-sigs/kube-scheduler-simulator/pull/81/commits/f80e7f2912890ca060fd7d772010e59013e2db02#diff-55b111ca13380e4133d565d2e89a7e59ba9dba0361de293248d0d05192a8236fR16